code-quality/accessibility/README.md

```diff
@@ -0,0 +1,74 @@
+# Accessibility
+
+## What is it?
+
+Accessibility is basically making sure web sites care fully inclusive and can be properly access for **everyone**, not just people with disabilities.
+
+The main standard for web accessibility is the Web Content Accessibility Guidelines (WCAG). The main standard is [WCAG 2.0](https://www.w3.org/TR/WCAG20/), but [WCAG 2.1](https://www.w3.org/TR/WCAG21/) came out in June 2018 and will be the main standard going forward.
+
+## Why?
+
+Various governments have accessibility requirements going forwards. Some countries, like Australia have a compliance requirement for all content. Europe has updates its directive requiring compliance on all public sector (non-government) site or risking fines or legal penalties. In the USA, there are accessibility requirements for Information and Communications Technology (ICT) that was updated in January of 2017 in [this article about section 508 of the Rehibilitiation Act](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/overview-of-the-final-rule). This only applies to federal, state, and local government sites. Public sector sites in the US are covered by Title III of the Americans with Disabilities Act. This title has not been updated to include websites.... *yet*.
```

Comment:
`...Europe has updated its direction...` what do you think?

Reply:
`...sites. These directions impose fines and/or legal penalties for non-compliance...`

Reply:
`...our government has imposed accessibility requirements...`

Reply:
`...Technology (ICT) that were updated....`

Reply:
English is hard.

Reply:
Actually, `directive` is intentional, since I'm referencing an actual government order. Everything else here I'm going with.